docs/pages/database-access/guides/postgres-redshift.mdx

```diff
@@ -175,6 +175,8 @@ $ tsh db logout my-redshift
 
 (!docs/pages/includes/database-access/aws-troubleshooting-max-policy-size.mdx!)
 
+(!docs/pages/includes/database-access/pg-cancel-request-limitation.mdx!)
```

Comment:
does it work for Redshift? same tables for ID?

Reply:
ah, no. Redshift you can query `select * from pg_stat_activity;` and it has a `procpid` column, but the query string is not available and the other columns aren't the same either.
Instead:
```sql
select pid, starttime, duration,
trim(user_name) as user,
trim (query) as querytxt
from stv_recents
where status = 'Running';
    pid     |         starttime          | duration |        user        |                 querytxt                  
------------+----------------------------+----------+--------------------+-------------------------------------------
 1073750177 | 2024-02-20 21:38:42.927815 | 14778363 | IAMR:GavinRedshift | select count(*)                          +
            |                            |          |                    | from generate_series(1, 10000000) AS s(i)+
            |                            |          |                    | where md5(i::text) not like '0%';
(1 row)
```

Redshift doesn't have `pg_sleep`, so I simulated a long running query like this:
```sql
select count(*)
from generate_series(1, 10000000) AS s(i)
where md5(i::text) not like '0%';
```

- `cancel <pid>;` works
- `select pg_cancel_backend(<pid>);` works
- ctrl+c works
- `select pg_terminate_backend(<pid>);` worked, but also made my other psql session hang (because it killed it) - seems redshift uses the same pid for my query and my session (or something idk really).

I'll update info for getting the pid

Reply:
[672fcfd](https://github.com/gravitational/teleport/pull/38352/commits/672fcfd6be5d24cad838eb5971ae1e5a0b5ce98d)